dev-util/radare2/radare2-5.5.4.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 toolchain-funcs
+
+DESCRIPTION="unix-like reverse engineering framework and commandline tools"
+HOMEPAGE="http://www.radare.org"
+
+ARM64_COMMIT=3c5eaba46dab72ecb7d5f5b865a13fdeee95b464
+ARMV7_COMMIT=dde39f69ffea19fc37e681874b12cb4707bc4f30
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/radareorg/radare2"
+else
+	SRC_URI="https://github.com/radareorg/radare2/archive/${PV}.tar.gz -> ${P}.tar.gz
+		https://github.com/radareorg/vector35-arch-arm64/archive/${ARM64_COMMIT}.tar.gz -> ${P}-vector35-arm64.tar.gz
+		https://github.com/radareorg/vector35-arch-armv7/archive/${ARMV7_COMMIT}.tar.gz -> ${P}-vector35-armv7.tar.gz
+	"
+
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="ssl"
+
+# Need to audit licenses of the binaries used for testing
+RESTRICT="test"
```

Comment:
Are you able to get tests working locally? This situation is obviously not ideal, but ensuring tests work for at least yourself would be great.

Reply:
Well, kind of, it works if I first emerge emerge radare2 and then I execute the tests. I don't know if it's possible to do an src_install before src_test is executed so all needed libs and binaries and in the correct paths, changing the path to include all stuff seems very ugly.

Reply:
> I don't know if it's possible to do an src_install before src_test is executed so all needed libs and binaries and in the correct paths, changing the path to include all stuff seems very ugly.

Yes, you can use the "${BUILD_DIR}" most likely for that. It's somewhat commonly used, the python eclasses even has built-in helpers for that.